src/battle-log.ts

```diff
@@ -760,6 +762,10 @@ class BattleLog {
 						attribs[styleValueIndex] = attribs[styleValueIndex] ?
 							Dex.getItemIcon(iconAttrib[1]) + '; ' + attribs[styleValueIndex] :
 							Dex.getItemIcon(iconAttrib[1]);
+					} else if (iconAttrib[0] === 'type') {
+						tagName = Dex.getTypeIcon(iconAttrib[1]).replace(/[<>]/g, '');
+					} else if (iconAttrib[0] === 'category') {
+						tagName = Dex.getCategoryIcon(iconAttrib[1]).replace(/[<>]/g, '');
```

Comment:
This is soooooooooo hacky and there's probably a better way to do this, but it works. Please let me know what the better way is, if there is one.

Reply:
I would `.slice(1, -3)`.

But I mean otherwise I suppose if it works, it's much easier than doing it any other way.